frontend/src/components/files/ListingItem.vue

```diff
@@ -270,4 +271,14 @@ const click = (event: Event | KeyboardEvent) => {
 const open = () => {
   router.push({ path: props.url });
 };
+
+const getExtension = (fileName: string): string => {
+  const lastDotIndex = fileName.lastIndexOf('.');
+  if (lastDotIndex === -1) {
+    return '';
```

Comment:
I think in that case we can return filename itself to support files like Dockerfile, Makefile etc

Reply:
Yes, you are right